src/components/media-player/ha-media-player-browse.ts

```diff
@@ -564,6 +573,48 @@ export class HaMediaPlayerBrowse extends LitElement {
     this._resizeObserver.observe(this);
   }
 
+  /**
+   * Load thumbnails for images on demand as they become visible.
+   */
+  private async _attachInteractionObserver(): Promise<void> {
+    if (!this._thumbnails) {
+      return;
+    }
+    if (!this._interactionObserver) {
+      this._interactionObserver = new IntersectionObserver(
+        async (entries, observer) => {
+          await Promise.all(
+            entries.map(async (entry) => {
+              if (!entry.isIntersecting) {
+                return;
+              }
+              const thumbnailCard = entry.target as HaCard;
+              let thumbnailUrl = thumbnailCard.dataset.src;
+              if (!thumbnailUrl) {
+                return;
+              }
+              if (thumbnailUrl.startsWith("/")) {
+                // Thumbnails served by local API require authentication
+                thumbnailUrl = await getSignedThumbnailPath(
+                  this.hass,
+                  thumbnailUrl
+                );
+              }
+              thumbnailCard.style =
+                "background-image:url(" + thumbnailUrl + ")";
+              observer.unobserve(thumbnailCard); // loaded, so no need to observe anymore
+              this.requestUpdate();
```

Comment:
Why are you requesting an update? you've already patched the element with the right CSS.

Reply:
I tested this without this line and it worked fine. Removed.